README.adoc

```diff
@@ -18,6 +18,22 @@
 //
 # MicroProfile OpenTracing
 
+'''
+== How to contribute
+
+Do you want to contribute to this project? Here is what you can do:
+
+* Fork the repository, make changes, then do a pull request.
+* https://github.com/eclipse/microprofile-opentracing/issues[Create or fix an issues].
+* https://gitter.im/eclipse/microprofile-opentracing[Join us on Gitter to discuss this project].
+* Join our https://calendar.google.com/calendar/embed?src=gbnbc373ga40n0tvbl88nkc3r4%40group.calendar.google.com[bi-weekly meeting] every second Wednesday at https://www.timeanddate.com/time/map/[16h00 GMT]. 
+** https://docs.google.com/document/d/1TbeKThAd9Df6IS0S6lz_YsTmxpFqWRmCL2HjbrAyLPQ/edit#heading=h.t9s5gp6jb2g6[Minutes and Agenda].
+** https://eclipse.zoom.us/j/949859967[Meeting room].
+* Join the discussions on the https://groups.google.com/forum/#!forum/microprofile[MicroProfile Google Group]
```

Comment:
Keep the links here but remove duplicated links from section below

Reply:
Done